### PR TITLE
Add 25 descriptions

### PR DESCRIPTION
**Pull request by rasa**
These are the last manifests in main without descriptions.

Descriptions pulled from https://github.com/ScoopInstaller/Main/pull/291/files#diff-49

### CODE REVIEW

File: bucket/imagemagick.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "https://www.imagemagick.org/script/index.php",
+    "description": "Create, edit, compose, and convert bitmap images, by reading and writing images in a variety of formats (over 200) including PNG, JPEG, GIF, HEIC, TIFF, DPX, EXR, WebP, Postscript, PDF, and SVG.",
```
Comment (Ash258) [inline bucket/imagemagick.json]:
```suggestion
    "description": "Create, edit, compose, and convert 200+ of bitmap images formats.",
```

File: bucket/kotlin-native.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "https://kotlinlang.org/",
+    "description": "Kotlin/Native is a technology for compiling Kotlin code to native binaries, which can run without a virtual machine.",
```
Comment (Ash258) [inline bucket/kotlin-native.json]:
```suggestion
    "description": "An LLVM backend for the Kotlin compiler, runtime implementation, and native code generation facility using the LLVM toolchain for native binaries runnable without virtual machine.",
```

File: bucket/haxe.json [hunk 3]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "https://haxe.org/",
+    "description": "An open source toolkit based on a modern, high level, strictly typed programming language, a cross-compiler, a complete cross-platform standard library and ways to access each platform's native capabilities.",
```
Comment (Ash258) [inline bucket/haxe.json]:
```suggestion
    "description": "High-level, cross-platform, multi-paradigm programming language and compiler.",
```

File: bucket/kubectl.json [hunk 4]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "https://kubernetes.io/docs/user-guide/kubectl-overview/",
+    "description": "A command-line tool for running commands against Kubernetes clusters.",
```
Comment (Ash258) [inline bucket/kubectl.json]: https://kubernetes.io/docs/reference/kubectl/kubectl/

```suggestion
    "description": "Control the Kubernetes cluster manager.",
```
or

```suggestion
    "description": "Kubernetes cluster manager controler.",
```

File: bucket/make.json [hunk 5]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "https://www.gnu.org/software/make/",
+    "description": "A tool which controls the generation of executables and other non-source files of a program from the program's source files.",
```
Comment (Ash258) [inline bucket/make.json]: I think this could be enough, but feel free to skip it.

```suggestion
    "description": "Build automation tool.",
```

**Comment by rasa** [comment]
Sorry about dismissing your reviews. GitHub does that automatically when I make any change. Can someone review this again so it can get merged in? Thanks!

**Pull request merged**